Makefile

```diff
@@ -125,7 +125,10 @@ prow-cluster-jobs:
 	oc create configmap prow-job-cluster-launch-installer-console --from-file=ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml -o yaml --dry-run | oc apply -f -
 	oc create configmap prow-job-cluster-scaleup-openshift-ansible-e2e --from-file=ci-operator/templates/openshift/openshift-ansible/cluster-scaleup-e2e-40.yaml -o yaml --dry-run | oc apply -f -
 	oc create configmap prow-job-master-sidecar-4.2 --from-file=ci-operator/templates/master-sidecar-4.2.yaml -o yaml --dry-run | oc apply -f -
+	oc create configmap prow-job-master-sidecar-4.2-s390x --from-file=ci-operator/templates/master-sidecar-4.2-s390x.yaml -o yaml --dry-run | oc apply -f -
```

Comment:
I don't think these sidecars are needed for the s390x/ppc enablement for the release controller. Would you agree @smarterclayton or am I missing something here

Reply:
I would love to know the answer to this, too, @smarterclayton

Reply:
I would guess that sidecar arch depends on where we are running the test suite in the CI cluster (which is all amd64 now), and is orthogonal to the arch of the cluster under test.  So that's another non-authoritative vote for "we can drop this line" ;)

Reply:
go ahead and drop these lines about the sidecar

Reply:
right on

Reply:
Alight. I deleted the commits for the sidecar stuff.